thirdparty/KDDockWidgets/src/private/Utils_p.h

```diff
@@ -12,7 +12,7 @@
 #ifndef KD_UTILS_P_H
 #define KD_UTILS_P_H
 
-#include "Config.h"
```

Comment:
We shouldn't make changes in third-party libraries in our repository, because when we update the library, we replace our version with the new version and we may lose our changes

Reply:
No code style fixes either I guess.

Reply:
In general, yes, third party libraries do not need to be changed. But in this case, I think, the changes are not critical, they can be kept, they will not interfere with the update to the new version.

Reply:
Hmm, @igorkorsukov merged it anyhow?!